diff --git a/lib/galaxy/web/base/controller.py b/lib/galaxy/web/base/controller.py
index 68d4753..6554b43 100644
--- a/lib/galaxy/web/base/controller.py
+++ b/lib/galaxy/web/base/controller.py
@@ -240,6 +240,8 @@ class JSAppLauncher( BaseUIController ):
     USER_BOOTSTRAP_KEYS = ( 'id', 'email', 'username', 'is_admin', 'tags_used', 'requests',
                             'total_disk_usage', 'nice_total_disk_usage', 'quota_percent' )
 
+    USER_BOOTSTRAP_KEYS = USER_BOOTSTRAP_KEYS + ('is_project_admin',)
+
     def __init__( self, app ):
         super( JSAppLauncher, self ).__init__( app )
         self.user_manager = users.UserManager( app )
